src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1583,26 +1585,58 @@ private void validate_nuspec(string nuspecFilePath, ChocolateyConfiguration conf
 
             if (!config.PackCommand.PackThrowOnUnsupportedElements)
             {
-                results = results.Where(r => !r.Message.contains("not supported"));
+                results = results.Where(r => !r.Id.StartsWith("CHCD") && !r.Id.StartsWith("CHCU"));

```

Comment:
Same question as above?  Should these be moved to a shared place, rather than putting them in three places?

Reply:
I have extracted out the logic into separate extension method that can be used (do note that this line you commented on is different than the others).

Reply:
Yes, understood.  It was the repetition of the usage of `CHCD`, and `CHCU` that I was really referring to.

Reply:
The usage of these strings is now contained into one location, so I am happy to move forward with this PR.